var/spack/repos/builtin/packages/fbgemm/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Fbgemm(CMakePackage):
+    """FBGEMM (Facebook GEneral Matrix Multiplication) is a low-precision,
+    high-performance matrix-matrix multiplications and convolution library
+    for server-side inference."""
+
+    homepage = "https://github.com/pytorch/FBGEMM"
+    git      = "https://github.com/pytorch/FBGEMM.git"
+
+    maintainers = ['dskhudia']
```

Comment:
@dskhudia Fyi, let me know if you agree with that (this means we'll ping you for review when there will be changes to FBGEMM `package.py` recipe)

Reply:
@alalazo see https://github.com/pytorch/FBGEMM/issues/629#issuecomment-861065409